plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/job/PodLogsJob.java

```diff
@@ -160,10 +160,10 @@ protected void runOCBinary(MultiStatus multiStatus) {
 			final MessageConsoleStream os = console.newMessageStream();
 			os.setEncoding("UTF-8");
 			try {
-				final InputStream logs = new BufferedInputStream(capability.getLogs(true, key.container, OpenShiftBinaryOption.SKIP_TLS_VERIFY));
-				byte [] data = new byte [8192];
+				final InputStream logs = capability.getLogs(true, key.container, OpenShiftBinaryOption.SKIP_TLS_VERIFY);
```

Comment:
Stack trace shows that there is already a BufferedInputStream in the chain of streams :). Anyway, if we read into array, it is by itself a buffer. Performance is affected only when we read by one byte.